.pre-commit-config.yaml

```diff
@@ -285,14 +285,14 @@ repos:
         language: system
         entry: "./scripts/ci/pre_commit_mypy.sh"
         files: \.py$
-        exclude: ^airflow/_vendor/.*$
+        exclude: ^airflow/_vendor/.*$|^dev
```

Comment:
?

Reply:
I found out that on linux when running pre-commit --all-files will fail without MOUNT_ALL_SOURCES (this is how it runs on Travis.

Reply:
Oh right. Was just wondering where this file/folder came from as I didn't see it in our repo anywhere.